misk-action-scopes/api/misk-action-scopes.api

```diff
@@ -14,13 +14,14 @@ public abstract class misk/concurrent/WrappingListeningExecutorService : com/goo
 	protected abstract fun wrap (Ljava/util/concurrent/Callable;)Ljava/util/concurrent/Callable;
 }
 
-public final class misk/scope/ActionScope : java/lang/AutoCloseable {
+public final class misk/scope/ActionScope : misk/scope/Scope {
 	public static final field Companion Lmisk/scope/ActionScope$Companion;
 	public final fun asContextElement ()Lkotlin/coroutines/CoroutineContext$Element;
 	public fun close ()V
-	public final fun enter (Ljava/util/Map;)Lmisk/scope/ActionScope;
+	public fun enter (Ljava/util/Map;)Lmisk/scope/Scope;
```

Comment:
Why is this being renamed? `ActionScope` is in use in a ton of places; not just code, but documentation. Its also a _good name_, it tells you that the scope of the thing is the `Action`. The new name does not tell you _what_ the scope is, and implies that it could be _any scope_.

Reply:
`Scope` is the new interface which will have a corresponding Armeria implementation to the `ActionScope` Misk implementation. I agree though, on first glance a confusing change.